hardhat/README.md

```diff
@@ -59,15 +59,29 @@ but they should be similar for other Linux versions and other platforms.
 
 ### Start an Optimistic Ethereum Node
 
-This process downloads, compiles, and builds an Optimistic Ethereum network. Note that it takes a long time.
+This process downloads and starts an Optimistic Ethereum network of one node.
 
 1. Clone the [Optimism monorepo](https://github.com/ethereum-optimism/optimism).
 
    ```sh
    git clone https://github.com/ethereum-optimism/optimism.git
-   cd optimism
+   cd optimism/ops
    ```
-   
+
+2. Start the Optimistic Ethereum node. This process downloads the images
+   from [the Docker hub](https://hub.docker.com/u/ethereumoptimism), and it
+   typically takes about ten minutes.
```

Comment:
It's arguable how long this takes, on my machine it is much lower than 10 min, probably around 3. Maybe say that depending on hardware this might take 10 min?